### ISSUE TITLE
Error in the log

### ISSUE THREAD

**Issue by mxm199** [imagegenius/docker-immich#367]
Good afternoon, tell me what the error is and is there any way to fix it?
![image](https://github.com/imagegenius/docker-immich/assets/133875469/002ec3fe-8ec5-4209-886a-d8040c203f3b)
These are the errors -
INFO Error while closing socket [Errno 9] Bad file descriptor
ERROR    Worker (pid:222) was sent SIGINT!

Otherwise, everything works normally, there are no other errors

**Comment by martabal** [comment]
Should be fixed with #356

**Comment by mxm199** [comment]
Hmm... Updated to the latest version

![image](https://github.com/imagegenius/docker-immich/assets/133875469/b30a8a34-bb52-4163-be81-6bfa912d111b)

Now the same error is from another service

**Comment by hydazz** [comment]
are you using this image or the official images?

**Comment by mxm199** [comment]
Yes, of course. I'm using it now -
image: ghcr.io/imagegenius/immich:latest
![image](https://github.com/imagegenius/docker-immich/assets/133875469/675f239e-2e9e-4610-a897-1d51edf6bfd2)

But there were such mistakes on other images too - 
image: ghcr.io/immich-app/immich-server:release
image: ghcr.io/immich-app/immich-machine-learning:release

**Comment by hydazz** [comment]
if the issue persists in the upstream images, please open an issue there as its an upstream issue 🙂

**Comment by mxm199** [comment]
Hmm, I launched the images from the main branch of the latest version 1.106.4 (I have deployed both branches and the AIO and the main branch and manually (not export-import) configured identically). And there are no errors in the main branch yet, I manually started all the tasks, scanned the library, everything is OK. I do not know at what point an error occurs on the AIO.

![image](https://github.com/imagegenius/docker-immich/assets/133875469/4f14535e-d0c0-44ae-b2f0-4aca1e98267f)

Of course, I will still test the version of the main branch, maybe an error will come out, but maybe someone has already come across the same error ...?
